src/js/components/HeaderMenu.vue

```diff
@@ -16,10 +30,22 @@
 
 <script>
 import { mapGetters, mapMutations } from 'vuex'
+import sqeManifest from '../../../sqe-manifest.json'
```

Comment:
you should be able to do `import sqeManifest from '@/sqe-manifest.json'.

Reply:
Thank you!